docs/indepth/installation.rst

```diff
@@ -35,13 +35,14 @@ On Fedora 20 you can install these by running:
 
 .. code-block:: console
 
-   alice@mercury:~$ sudo yum install @buildsys-build python python-devel python-virtualenv
+   alice@mercury:~$ sudo yum install @buildsys-build python python-devel python-virtualenv libffi-devel openssl-devel
```

Comment:
At least for the two in this document we could make a `task` if you think that is worthwhile.
That would also be necessary if we are ever going to test this (which we now have a framework for).